RxCocoa/Common/DelegateProxy.swift

```diff
@@ -225,19 +225,37 @@
         }
 
         override open func responds(to aSelector: Selector!) -> Bool {
-            return super.responds(to: aSelector)
+            if  aSelector == #selector(self.isDelegateProxy(_:)) {
+                return true;
+            }
+
+            let res = super.responds(to: aSelector)
```

Comment:
No need to store this into a value, just return directly

Reply:
You are right